certified-connectors/Alkymi/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1301 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Alkymi",
+    "description": "Programmatic interface for the Alkymi Data Inbox",
+    "contact": {
+      "name": "Alkymi Support",
+      "url": "https://www.alkymi.io/contact",
+      "email": "tori@alkymi.io"
+    }
+  },
+  "host": "www.example.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/api/v2/datasets/": {
+      "get": {
+        "description": "Lists all of the datasets available on the tenant.\\nhttps://alkymi-api.readme.io/reference#get-dataset-list",
+        "summary": "List All Datasets",
+        "tags": [
+          "datasets"
+        ],
+        "operationId": "datasets_list",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "",
```

Comment:
@tori-alkymi Please provide a valid description value for the response code, and you can all the possible response codes of the operation.

Applies the same comment for all the similar properties in the swagger.